bus-mapping/src/evm/opcodes.rs

```diff
@@ -336,27 +338,38 @@ pub fn gen_begin_tx_ops(state: &mut CircuitInputStateRef) -> Result<ExecStep, Er
         },
     )?;
 
-    match (call.is_create(), state.is_precompiled(&call.address)) {
-        (true, _) => {
-            // TODO: Implement creation transaction
-        }
-        (_, true) => {
-            // TODO: Implement calling to precompiled
-        }
-        _ => {
-            state.push_op(
-                &mut exec_step,
-                RW::READ,
-                AccountOp {
-                    address: call.address,
-                    field: AccountField::CodeHash,
-                    value: code_hash.to_word(),
-                    value_prev: code_hash.to_word(),
-                },
-            );
-        }
+    // There are 4 branches from here.
+
+    // 1. Creation transaction.
+    if call.is_create() {
+        // TODO:
+        return Ok(exec_step);
```

Comment:
I was thinking to just pass instead of panic to make integration-test work. I have added `log::warn` as Edu suggested in https://github.com/appliedzkp/zkevm-circuits/pull/278#discussion_r838360862.

Reply:
That's perfect!!